### PR TITLE
main section creation

### PR DESCRIPTION
**Pull request by AugustoTI**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
```
Comment (JonathanFerreirar) [inline .editorconfig:1]: this is amazing you saved us, adding that.🥰

File: src/Router.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import React from "react";
+import React from 'react'
```
Comment (JonathanFerreirar) [inline src/Router.js:1]: This made a lot of unnecessary changes, you must have set it to use "(double quotes) and not ' (single quotes), you made a lot of useless changes.

File: src/Router.js [hunk 3]
```diff
@@ -14,10 +15,11 @@ const Router = () => {
...
+          <Route path="/section-main" element={<SectionMain />} />
         </Routes>
       </BrowserRouter>
     </div>
-  );
-};
+  )
+}
 
-export { Router };
```
Comment (JonathanFerreirar) [inline src/Router.js:23]: It is much better to use semicolons, it makes the code easier to read by making it obvious where the code ends.

File: src/Router.js (hunk 2 above)
Comment (AugustoTI) [reply src/Router.js:1]: Well, that was my extension in Prettier at the time I applied it, but the correct thing would be to have defined this in the root of the project in a configuration file to standardize the code formatting

File: src/global.css [hunk 4]
```diff
@@ -1,98 +1,96 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
-
```
Comment (JonathanFerreirar) [inline src/global.css:6]: No need say, nothing....

File: tailwind.config.js [hunk 5]
```diff
@@ -1,25 +1,91 @@
 /** @type {import('tailwindcss').Config} */
```
Comment (JonathanFerreirar) [inline tailwind.config.js:1]: No need say, nothing....

File: src/laddingpage/sections/Main.jsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { Button } from '../../components/button/Button'
```
Comment (JonathanFerreirar) [inline src/laddingpage/sections/Main.jsx:1]: We are using .js and this project and not .jsx, pay attention on the desing pattern.
